ui/app/src/Silences/Types.elm

```diff
@@ -37,6 +44,12 @@ type alias Silence =
     , updatedAt : Time
     , matchers : List Matcher
     , silencedAlerts : ApiData (List Alert)
+    , status : SilenceStatus
+    }
+
+
+type alias SilenceStatus =
```

Comment:
Actually, looking at this, is there a reason it isn't a union type?

```elm
type Status
    = Active
    | Active
    | Pending
```

I think I would prefer this, as it saves us from having to do string matching later to check the status.